target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -196,6 +196,7 @@ ramips_setup_interfaces()
 		;;
 	ar670w|\
 	ar725w|\
+	rt-ac54u|\
```

Comment:
asus,rt-ac54u

Reply:
other asus devices not add "asus,xxx"

I reject